### PR TITLE
Add to the password story option when user decides not to enter the maze

### PR DESCRIPTION
**Pull request by trthhrtz**

### CODE REVIEW

**Comment by trthhrtz** [comment]
Please review this, @casey-collab

File: english/plugh/outside/outside1.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+You decided to rest a little bit. 
+
+Password is still in your pocket. Weather is nice, birds are tweeting. 
+
+*Pstst*. Can of soda is opened. You drink slowly and enjoy it. 
+
+However your inner voice stars raising up. You think you noticed some movement. 
+
+[Look around](outside/outside1/outside.md)
```
Comment (casey-collab) [inline english/plugh/outside/outside1.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: english/plugh/outside/outside1.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+[Keep drinking](outside/outside2/outside.md)
```
Comment (casey-collab) [inline english/plugh/outside/outside1.md]: This link is also invalid.

Comment (casey-collab) [inline english/plugh/outside/outside1.md]: There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.

File: english/plugh/outside/outside1/outside.md [hunk 3]
```diff
@@ -0,0 +1 @@
+And ....
```
Comment (casey-collab) [inline english/plugh/outside/outside1/outside.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

File: english/plugh/outside/outside2/outside.md [hunk 4]
```diff
@@ -0,0 +1 @@
+And....
```
Comment (casey-collab) [inline english/plugh/outside/outside2/outside.md]: This page is also unreachable.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by trthhrtz** [comment]
Please review this, @casey-collab

File: english/plugh/outside/outside1.md [hunk 5]
```diff
@@ -0,0 +1,10 @@
+You decided to rest a little bit. 
+
+Password is still in your pocket. Weather is nice, birds are tweeting. 
+
+*Pstst*. Can of soda is opened. You drink slowly and enjoy it. 
+
+However your inner voice stars raising up. You think you noticed some movement. 
+
+[Look around](outside1/outside.md)
+[Keep drinking](outside2/outside.md)
```
Comment (casey-collab) [inline english/plugh/outside/outside1.md]: There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by trthhrtz** [comment]
Please review this, @casey-collab

**Comment by casey-collab** [comment]
Looks great! :melon:

**Pull request merged**